containers/unit-tests/Dockerfile

```diff
@@ -1,38 +1,22 @@
-FROM ubuntu:16.04
+ARG debian_arch=amd64
 
-ARG arch=amd64
+FROM ${debian_arch}/ubuntu:19.04
 
-# dependencies which must be installed for the target architecture
-# we must do cross-builds on i386 as phantomjs is not available for i386
-ARG _crossdeps="libglib2.0-dev libgudev-1.0-dev libjson-glib-dev libkeyutils-dev liblvm2-dev libnm-glib-dev \
-    libpam0g-dev libpcp3-dev libpcp-import1-dev libpcp-pmda3-dev libpolkit-agent-1-dev libpolkit-gobject-1-dev \
-    libssh-dev libsystemd-dev libkrb5-dev libgnutls28-dev \
-    glib-networking glib-networking-dbg libc6-dbg libglib2.0-0-dbg pkg-config"
+ARG personality=linux64
+ADD setup.sh /
+RUN ${personality} /setup.sh ${personality}
+RUN rm -rf /setup.sh
```

Comment:
Can you explain this comment?

Reply:
docker images are organized in layers, each of which is a file diff towards the previous one. The first one is FROM, and then each RUN or ADD command adds a new delta. This command creates a new layer with a delta which removes setup.sh. Thus instead of saving a few bytes, you add a new layer and setup.sh is still present in the previous layer.